post-scan-actions/aws-python-promote-or-quarantine/README.md

```diff
@@ -1,322 +1,60 @@
     - Scroll down to locate the  **ScanResultTopic** Logical ID.
     - Copy the **ScanResultTopic** ARN to a temporary location. Example: `arn:aws:sns:us-east-1:123445678901:FileStorageSecurity-All-In-One-Stack-StorageStack-1IDPU1PZ2W5RN-ScanResultTopic-N8DD2JH1GRKF`
-2. **Find the Lambda function ARN**
 
-    📌 The Lamdba function ARN is only required if you plan to use the AWS CLI (as opposed to the console) to subscribe the Lambda to the SNS topic.
-    - In the AWS console, go to **Services > Lambda**.
-    - Search for the Lambda function you created previously. Example: `FSS_Prom_Quar_Lambda`
-    - Click the Lambda function link.
-    - On the top-left, locate the **ARN**.
-    - Copy the ARN to a temporary location. Example: `arn:aws:lambda:us-east-1:123445678901:function:FSS_Prom_Quar_Lambda`
-3. **Subscribe the Lambda to the SNS topic**
+## Installation
 
-    <details>
-    <summary>Using the AWS console</summary>
+### From AWS Lambda Console
 
-    1. Go to **Services > Lambda**.
-    2. Search for the Lambda function you created previously. Example: `FSS_Prom_Quar_Lambda`
-    3. Click the link to your Lambda function to view its details.
-    4. Click **Add trigger** on the left.
-    5. From the **Trigger configuration** list, select **SNS**.
-    6. In the **SNS topic** field, enter the SNS topic ARN you found earlier.
-    7. Click **Add**. Your Lambda is now subscribed to the SNS topic.
+1. Visit [the app's page on the AWS Lambda Console](https://console.aws.amazon.com/lambda/home?region=us-east-1#/create/app?applicationId=arn:aws:serverlessrepo:us-east-1:415485722356:applications/cloudone-filestorage-plugin-action-promote-or-quarantine).
+2. Fill in the parameters.
+3. Check the `I acknowledge that this app creates custom IAM roles.` checkbox.
+4. Click `Deploy`.
 
-    </details>
+### Embed as a Nested App in Your Serverless Application
 
-    <details>
-    <summary>Using the AWS CLI</summary>
+1. Visit [the app's page on the AWS Lambda Console](https://console.aws.amazon.com/lambda/home?region=us-east-1#/create/app?applicationId=arn:aws:serverlessrepo:us-east-1:415485722356:applications/cloudone-filestorage-plugin-action-promote-or-quarantine).
+2. Click the `Copy as SAM Resource` button and paste the copied YAML into your SAM template, filling in any required parameters.
 
-    - Grant the Lambda function permissions to attach a trigger via AWS CLI command:
+### Other Options
 
-        ```bash
-        aws lambda add-permission --function-name <YOUR_FSS_FUNC_NAME> \
-        --source-arn <SNS_TOPIC_ARN> \
-        --statement-id <YOUR_FSS_FUNC_NAME> --action "lambda:InvokeFunction" \
-        --principal sns.amazonaws.com
-        ```
-    - Enter the following AWS CLI command to subscribe your Lamdba function to the SNS topic:
+Checkout [other installation methods](./other-installation-methods.md), such as AWS CLI or SAM CLI.
```

Comment:
Check or checkout?
```suggestion
Check [other installation methods](./other-installation-methods.md), such as AWS CLI or SAM CLI.
```

Reply:
Maybe `Check out`? Leaving this to @dougc-trend

Reply:
Check out [other installation methods](./other-installation-methods.md), such as AWS CLI or SAM CLI.